src/backend/distributed/citus--7.1-3--7.1-4.sql

```diff
@@ -0,0 +1,54 @@
+/* citus--7.1-3--7.1-4 */
+
+CREATE TYPE citus.shard_transfer_mode AS ENUM (
+   'require_replica_identity',
+   'force_logical',
```

Comment:
I don't have any strong opinions, but, I was thinking that we'd use `force_logical_replication` which is more explicit. That said, feel free to ignore this comment.

Reply:
I slightly prefer `force_logical` partly because I can't type words that long :).